docs/examples/vectorfields.py

```diff
@@ -0,0 +1,493 @@
+# %% [markdown]
+# # Gaussian Processes for Vector Fields
+# In this notebook, we use Gaussian processes to learn vector valued functions. We will be
+# recreating the results by [Berlinghieri et. al, (2023)](https://arxiv.org/pdf/2302.10364.pdf).
+# %%
+from jax.config import config
+
+config.update("jax_enable_x64", True)
+import torch
+
+torch.manual_seed(123)
+import jax.random as jr
+import jax.numpy as jnp
+from jaxtyping import install_import_hook
+import matplotlib.pyplot as plt
+from matplotlib import rcParams
+from mpl_toolkits.axes_grid1 import make_axes_locatable
+import numpy as onp
+import optax as ox
+import tensorflow_probability as tfp
+from jax.config import config
+from dataclasses import dataclass
+from jaxtyping import (
+    Array,
+    Float,
+    install_import_hook,
+)
+from jax import hessian, jit
+
+with install_import_hook("gpjax", "beartype.beartype"):
+    import gpjax as gpx
+
+# Enable Float64 for more stable matrix inversions.
+key = jr.PRNGKey(123)
+plt.style.use(
+    "https://raw.githubusercontent.com/JaxGaussianProcesses/GPJax/main/docs/examples/gpjax.mplstyle"
+)
+cols = rcParams["axes.prop_cycle"].by_key()["color"]
```

Comment:
I would suggest to rename this to 
```suggestion
colors = rcParams["axes.prop_cycle"].by_key()["color"]
```
to make it less ambiguous - when reading `cols` further down where it's used, I first thought "columns"!